Makefile

```diff
@@ -63,6 +63,19 @@ docker-push: ## Push docker image with the operator.
 	@docker --config=$(DOCKER_CONF) push $(REPO):$(TAG)
 	@docker --config=$(DOCKER_CONF) push $(REPO):latest
 
+##@ Release
```

Comment:
Wondering why we need all these build targets? Seems this is simply the `build_tag.sh` implementation. It does not really matter but I think bash is easier to read than 3 makefile targets.

In any case thanks for the fix LGTM